cocotb/bfmgen.py

```diff
@@ -0,0 +1,364 @@
+###############################################################################
+# Copyright cocotb contributors
+# Licensed under the Revised BSD License, see LICENSE for details.
+# SPDX-License-Identifier: BSD-3-Clause
+###############################################################################
+import os
+import argparse
+import importlib
+from string import Template
+
+import cocotb
+from cocotb.bfms import BfmMgr
+
+
+def bfm_load_modules(module_l): 
+    for m in module_l:
+        try:
+            importlib.import_module(m)
+        except Exception as e:
+            print("Error: failed to load module \"" + str(m) + "\": " + str(e))
+            raise e
+
+def process_template_vl(template, info):
+    t = Template(template)
+    
+    bfm_import_calls = ""
+    for i in range(len(info.import_info)):
```

Comment:
```suggestion
    for i, imp in enumerate(info.import_info):
```
and remove the line below

Reply:
Done!